share/csm_share/shr/shr_orb_mod.F90

```diff
@@ -78,8 +78,10 @@ real(SHR_KIND_R8) pure FUNCTION shr_orb_cosz(jday,lat,lon,declin,dt_avg)
       shr_orb_cosz =  shr_orb_avg_cosz(jday, lat, lon, declin, dt_avg)
    else
       shr_orb_cosz = sin(lat)*sin(declin) - &
-           cos(lat)*cos(declin) * &
-           cos((jday-floor(jday))*2.0_SHR_KIND_R8*pi + lon)
+      &              cos(lat)*cos(declin)*cos(jday*2.0_SHR_KIND_R8*pi + lon)
```

Comment:
That was my suggestion for shr_flux_mod - and I agree that it should be applied in shr_orb_mod also.